appcues/src/main/java/com/appcues/ui/composables/StepComposition.kt

```diff
@@ -56,17 +56,21 @@ internal fun Step.ApplyOverlayStepTraits(boxScope: BoxScope, stepDecoratingPaddi
 }
 
 @Composable
-internal fun Step.ComposeStickyContent(boxScope: BoxScope, stepDecoratingPadding: StepDecoratingPadding) {
+internal fun Step.ComposeStickyContent(boxScope: BoxScope, contentPadding: PaddingValues?, stepDecoratingPadding: StepDecoratingPadding) {
     topStickyContent?.let {
         Box(
-            modifier = Modifier.alignStepOverlay(boxScope, Alignment.TopCenter, stepDecoratingPadding),
-            contentAlignment = Alignment.BottomCenter
+            modifier = Modifier
+                .then(if (contentPadding != null) Modifier.padding(contentPadding) else Modifier)
+                .alignStepOverlay(boxScope, Alignment.TopCenter, stepDecoratingPadding),
+            contentAlignment = Alignment.TopCenter
```

Comment:
note: this was `BottomCenter` before, on the top case, but seems it should match the alignment on the line above? in practice I don't think it really matters much, since the box height is the same as the sticky content height inside it here.

Reply:
yes it wont change much but its good to keep it matching the alignStepOverlay for consistency